File: corpus/javascript/fixture_22.js
export function calculateDiscount22(cartItems, taxRate) {
  const totalAmount = cartItems.reduce((sum, item) => sum + item.price * (item.qty ?? 1), 0);
  return totalAmount > 220 ? totalAmount * (1 + taxRate) : totalAmount;
}

export class PriceBook22 {
  lookupPrice(sku) { return sku.length + 22; }
}

File: corpus/go/fixture_22.go
package fixture22

type PriceBook22 struct {
    Multiplier float64
}

func CalculateDiscount22(cartItems []float64, taxRate float64) float64 {
    totalAmount := 0.0
    for _, price := range cartItems {
        totalAmount += price
    }
    if totalAmount > 220.0 {
        return totalAmount * (1.0 + taxRate)
    }
    return totalAmount
}
